ir/importer.py

```diff
@@ -252,6 +305,45 @@ def importPocket(self):
             mw.progress.finish()
             tooltip('Added {} item(s) to deck: {}'.format(n, deck))
 
+    def importEpub(self, epub_file_path = None):
+        if not epub_file_path:
+            epub_file_path  = getFile(None, 'Enter epub File path', None, filter="*.epub")
+
+        if not epub_file_path:
+            return
+
+        articles = get_epub_toc(epub_file_path)
+        if not articles:
+            return
```

Comment:
It'd be great to show the user a message to tell them we failed to parse the epub file.